aws-observability-terraform/source-module/outputs.tf

```diff
@@ -58,6 +58,26 @@ output "elb_auto_enable_stack" {
   description = "AWS CloudFormation stack for ALB Auto Enable access logs."
 }
 
+output "clasic_lb_sns_topic" {
+  value       = local.create_classic_lb_source && !var.classic_lb_source_details.bucket_details.create_bucket ? module.classic_lb_module["classic_lb_module"].aws_sns_topic : {}
+  description = "SNS topic created to be attached to an existing elb logs bucket."
```

Comment:
for classic elb outputs occurrences - refer it by classic elb or classic lb

Reply:
Done